### PR TITLE
Bump persona-php-client to >=3.1

### PR DESCRIPTION
**Pull request by camillef**

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
         <!-- Install dependencies through Composer -->
         <exec executable="php" failonerror="true">
             <arg value="composer.phar"/>
-            <arg value="install"/>
+            <arg value="update"/>
```
Comment (malcyL) [inline build.xml:30]: Is this in the init target? Why is it changing to update? Is that ok for the first time?

File: composer.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
     ],
     "require":{
         "php": ">=5.3.0",
-        "guzzle/guzzle":"3.8.1",
-        "talis/persona-php-client": "~2.0"
+        "guzzle/guzzle":">=3.8@stable",
+        "talis/persona-php-client": ">=3.1"
```
Comment (markwallsgrove) [inline composer.json:24]: should this  be more strict? something like `~3.1`? Doesn't `~` include all minor versions, but no major changes?

File: .travis.yml [hunk 3]
```diff
@@ -4,8 +4,6 @@ php:
 - 5.5
 - 5.4
 - 5.3
-
 install:
-- composer install
-
+- ant init
```
Comment (markwallsgrove) [inline .travis.yml:8]: I guess these changes work on a refresh checkout,  build machines, etc?

**Comment by markwallsgrove** [comment]
other than the comments above :+1:

File: build.xml (hunk 1 above)
Comment (camillef) [reply build.xml:30]: Just trying to make it consistent with echo-php-client

File: .travis.yml (hunk 3 above)
Comment (camillef) [reply .travis.yml:8]: Again - this makes it consistent with echo-php-client - and actually stopped it failing

**Pull request closed without merging**